### DISCUSSION TITLE
Default Location of Graph Data .json File

### DISCUSSION THREAD

**Discussion by manunamz** [manunamz/jekyll-wikilinks#16] [General]
Graph data is currently written to the `/assets` directory or a given directory in the `d3_graph_data` `assets_rel_path` config option.

Should the default write location be in jekyll's `_data/` dir instead?

**Comment by xplosionmind** [comment]
I think it makes sense. Even though now that its path is customizable in the config file I would set it to be in `/_includes`, so that I can include it in the JavaScript file I am writing to show the graph.
